django-stubs/contrib/admin/options.pyi

```diff
@@ -93,35 +107,35 @@ class BaseModelAdmin:
     def has_module_permission(self, request: HttpRequest) -> bool: ...
 
 class ModelAdmin(BaseModelAdmin):
-    list_display: Sequence[Union[str, Callable]] = ...
+    list_display: Sequence[Union[str, Callable[[Model], Any]]] = ...
```

Comment:
Can we reduce `Any` here? Or is it hard to do?

Reply:
I actually went for that at first, with a union of callables returning `str` or `bool` which are the two cases mentioned in the docs, but I now think this a case of the docs being vague. I think any type that is `str()`-able is a valid return type here, which is virtually any Python type, right? We could probably change to `object` here, not sure if that adds any value though.

Reply:
Ok, let's leave `Any`. Thanks a lot for the explanation!